Adding Courses for Hindi Language

Ravikisha opened the pull request:
## What does this PR do?
Add resource(s)

## For resources
### Description
Adding online free courses for the Hindi language

### Why is this valuable (or not)?
Learners will learn some courses in Hindi and some good YouTubers will be recognized. 

### How do we know it's really free?
All the resources are from YouTube and free websites. These courses will be free always.

### For book lists, is it a book? For course lists, is it a course? etc.
#### Courses
- Laravel
- Asp.Net
- React Native
- Game Development
- Spring Boot

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -10,11 +10,13 @@
 * [Compiler Design](#compiler-design)
 * [Computer Graphics](#computer-graphics)
 * [Computer Organization and Architecture](#computer-organization-and-architecture)
+* [Cyber Security](#cyber-security)
```
Most of our lists just use "Security"

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -146,6 +151,14 @@
+### <a id="asp.net"></a>ASP.NET
+
+* [ASP .Net in Hindi](https://www.youtube.com/playlist?list=PLbsXhdgwIKL1g2vE86yGBK_RYuqt98yWL) - ComputerHindiNotes
+* [ASP.NET CORE 6.0 TUTORIALS IN HINDI](https://www.youtube.com/playlist?list=PL18HZjtdIA4DiYGQj1zst6myBAVE3wgMg) - Technology Keeda
```
no ALL CAPS

eshellman replied:
shouldn't this one supercede the following one? Why both?

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -146,6 +151,14 @@
+
+* [ASP .Net in Hindi](https://www.youtube.com/playlist?list=PLbsXhdgwIKL1g2vE86yGBK_RYuqt98yWL) - ComputerHindiNotes
+* [ASP.NET CORE 6.0 TUTORIALS IN HINDI](https://www.youtube.com/playlist?list=PL18HZjtdIA4DiYGQj1zst6myBAVE3wgMg) - Technology Keeda
+* [ASP.NET CORE TUTORIALS IN HINDI](https://www.youtube.com/playlist?list=PL18HZjtdIA4Al-wYHC-i2TA-lgXLvHVmB) - Technology Keeda
```
no ALL CAPS

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -191,6 +204,15 @@
+* [cyber security course for beginners - Urdu/Hindi](https://www.youtube.com/playlist?list=PLKJfBg0XdWkevCEJ64RK11LylBNX2-zbk) - hashintelligence
+* [Cyber Security Tutorial In Hindi](https://www.youtube.com/playlist?list=PL-JvKqQx2AteIbm-z4X709scVr9OaHpIY) - University Academy
+* [Cyber Security Tutorials In HIndi](https://www.youtube.com/playlist?list=PL0fjgIGwLMWTFmZoLdEPoI9azA-osxcQj) - TechChip
+* [Ethical Hacking Full Couse || Cyber Security in Hindi](https://www.youtube.com/playlist?list=PLa2xctTiNSCibSUhgYI2RT_loUJP9rDN6) - Masters in IT
```
escape the `|`

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -315,7 +342,16 @@
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
 * [Java Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - CodeWithHarry
 * [Microservice Tutorial in Hindi 2023](https://www.youtube.com/playlist?list=PL0zysOflRCelb2Y4WOVckFC6B050BzV0D) - Learn Code With Durgesh
+* [Spring Framework Tutorial with SpringBoot | in Hindi| Complete Course | CoderX Ankit](https://www.youtube.com/playlist?list=PLiOIhBfKi8AIYNWEjYy1zZFe1N8GvPRHR) - CoderX Ankit
```
escape the `|`

eshellman replied:
```suggestion
* [Spring Framework Tutorial with SpringBoot  in Hindi Complete Course](https://www.youtube.com/playlist?list=PLiOIhBfKi8AIYNWEjYy1zZFe1N8GvPRHR) - CoderX Ankit
```

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -459,6 +504,16 @@
+### Laravel
+
+* [Laravel (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhijEqjCa63l0GkWh5EsG5iTR) - Geeky Shows
+* [Laravel 10 Tutorial Course in Hindi / Urdu](PL0b6OzIxLPbz7JK_YYrRJ1KxlGG4diZHJ) - Yahoo Baba
```
you forgot the link

eshellman reviewed (changes requested):
It's easier to get a PR approved if it makes fewer changes

Ravikisha commented:
**Update Requested Changes 👍**

Thank you for addressing the feedback provided in the initial review. I've reviewed the changes you made.

Feel free to reach out if you have any questions or need further clarification on any of these points. Let's work together to ensure the code meets your standards and requirements. Thanks for your effort!

Ravikisha commented:
I resolved the conflict and Thanks for your effort.

eshellman commented on courses/free-courses-hi.md:
```diff
@@ -373,7 +398,16 @@
+* [React Native Tutorial in Hindi](https://www.youtube.com/playlist?list=PL8p2I9GklV468O2wk-n8Q1KmtMhnHHj4C) - Code Step By Step
+* [React Native Tutorial In Hindi - 2023](https://www.youtube.com/playlist?list=PLvN7nvnjkvpQribRyQ4r0FYZxKsPLGciy) - Code Diggers
+* [React Native Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLwGdqUZWnOp354xMD8u0hxX-1qmCvfLiY) - Thapa Technical
+* [React Native Tutorials (mobile app development) in hindi](https://www.youtube.com/playlist?list=PLB97yPrFwo5gxB5SuNWzH73t2Su65KN2f) - CODERS NEVER QUIT
```
no ALL CAPS

Ravikisha commented:
Sorry for the issue.

eshellman commented:
Thanks, and welcome to free-programming-books! For future PRs, you can minimize conflicts by working on one section at a time.
